### ISSUE TITLE
Priorty Encoder Enable input is really a Valid output

### ISSUE THREAD

**Issue by nTrfAce** [CircuitVerse/CircuitVerse#1702]
**Describe the bug**
The documentation for the Priority Encoder https://docs.circuitverse.org/#/decodersandplexers?id=priority-encoder says "AN enable input is also provided to activate/deactivate the priority encoder."  The connection at the bottom of the symbol is labeled EN.

However, this is not an Enable input, rather it is a output which indicates if the output is Valid.  The Valid output goes high when there is at least one input which is high.  If the Valid output were not present it would be ambiguous if there was no input to the Priority Encoder, or if the lowest priority input was asserted.  Further the Valid output is very useful, as this circuit element can take an arbitrary number of interrupts in, and consolidate them to a single interrupt out.

https://en.wikipedia.org/wiki/Priority_encoder describes this as "The output V indicates if the input is valid."

**To Reproduce**
If you wire an input circuit element to EN and set it to 0, and an input circuit element to any of the Priority Encoder's inputs (0, 1, ...) and drive it to 1, you get "Contention Error: 0 and 1" and "Contention Error: 1 and 0".

**Expected behavior**
The circuit element is working properly, I think this is a documentation bug, that EN should be re-labeled to V, and the documentation updated, adding a V column to the table, a row to the table showing no input asserted, and the help page text "AN enable input is also provided to activate/deactivate the priority encoder." changed to "A 'Valid' output labeled V is provided to indicate that at least one input is 1.  Valid 0 means no valid input is present on any of the inputs, IE all inputs are 0.  Valid 1 means at least one input is 1.  Read the binary value of the output(s) to determine the input number of the highest input which is active.  An example use of this circuit element is to indicate if any interrupt is present to a system, and to provide the value of the highest priority interrupt which is asserted."

**Screenshots**
Here is a project which makes use of the mislabeled but functional "Valid" output signal:
![image](https://user-images.githubusercontent.com/25803511/94272279-b8430a80-fef7-11ea-8e80-6259de962a6a.png)
https://circuitverse.org/users/37662/projects/priority-encoder-to-hex-7-segment

Here is an image showing a minimal circuit to reproduce the contention error if it were an Enable input:
![priority encoder contention issue](https://user-images.githubusercontent.com/25803511/94272131-7e720400-fef7-11ea-8233-8a3fd2dd60aa.PNG)

**Are you working on this issue? (Yes/No)**
No

**Comment by satu0king** [comment]
Thanks for reporting this!

**Comment by nTrfAce** [comment]
The octocat waved to me from the corner of the docs page,
![image](https://user-images.githubusercontent.com/25803511/94349049-e0189800-fff5-11ea-8443-05d2dc870986.png)
leading me to realize that the docs are in markdown on github at:
https://github.com/CircuitVerse/CircuitVerseDocs/blob/master/docs/decodersandplexers.md
Cool.

The truth table edit I'm suggesting would be in markdown:
```
|   T3    |   T2    |   T1    |   T0    |    V    |   O2    |   O1    |
|---------|---------|---------|---------|---------|---------|---------|
|    0    |    0    |    0    |    0    |    0    |    0    |    0    |  
|    0    |    0    |    0    |    1    |    1    |    0    |    0    |  
|    0    |    0    |    1    |    X    |    1    |    0    |    1    |
|    0    |    1    |    X    |    X    |    1    |    1    |    0    |
|    1    |    X    |    X    |    X    |    1    |    1    |    1    |
```
Which shows as the table:
|   T3    |   T2    |   T1    |   T0    |    V    |   O2    |   O1    |
|---------|---------|---------|---------|---------|---------|---------|
|    0    |    0    |    0    |    0    |    0    |    0    |    0    |  
|    0    |    0    |    0    |    1    |    1    |    0    |    0    |  
|    0    |    0    |    1    |    X    |    1    |    0    |    1    |
|    0    |    1    |    X    |    X    |    1    |    1    |    0    |
|    1    |    X    |    X    |    X    |    1    |    1    |    1    |

**Comment by satu0king** [comment]
@nTrfAce, you are right. You can simply send in a PR yourself to fix the bug or wait for someone to fix the same. Thanks!

**Comment by nTrfAce** [comment]
Okay, I'll take this as "I'm working on this issue", with the intention of submitting a Pull Request with the documentation fix and code fix for EN to V and fixing 'enable' to 'valid' internally in the module.

**Comment by nTrfAce** [comment]
Module PR
https://github.com/CircuitVerse/CircuitVerse/pull/1718

Documentation PR
https://github.com/CircuitVerse/CircuitVerseDocs/pull/276

Someone with access please edit https://circuitverse.org/simulator/embed/762 to change the input box on the bottom of the circuit.  Remove the input box, have instead V run to an output box, and place it in the bottom right of the circuit.
